msproteomicstoolslib/format/SWATHScoringMapper.py

```diff
@@ -300,11 +301,10 @@ def sqlInferMapping(rawdata_files, aligned_pg_files, mapping, precursors_mapping
                     diskfile_base = diskfile_base.split(ending)[0]
 
                 # 2.2 remove common file endings from the tsv data
-                for ending in [".tsv", ".csv", ".xls", "_with_dscore_filtered", "_with_dscore", "_all_peakgroups"]:
+                for ending in [".gz", ".mzML", ".tsv", ".csv", ".xls", "_with_dscore_filtered", "_with_dscore", "_all_peakgroups"]:
```

Comment:
@grosenberger see https://github.com/PyProphet/pyprophet/issues/49 why is that ?

Reply:
We could change the default PyProphet export format again to ``legacy_merged``.